src/components/MetricsPane/MetricsPane.styles.js

```diff
@@ -12,8 +12,9 @@ export const StyledMetricsWrapper = styled.div`
   ${({ $showMetricsPane }) =>
     $showMetricsPane
       ? css`
-          min-width: 42rem;
-          max-width: 42rem;
+          min-width: 41rem;
+          width: 50vw;
```

Comment:
Im a bit weary about using viewport relative units in children and nested components. It may work now, but make things complicates with future changes to any layouts or spacing higher up. Would other relative units, like plain old `%` work here?

I sense `vw` and `vh` are more appropriate for root level layout elements.